### PR TITLE
Calculate number of unique for Unknown columns

### LINKED ISSUE #1257: Calculate nunique for Unknown columns in `_get_describe_dict`
- As a user, I wish I could use Woodwork's describe method to inspect the uniqueness of Unknown columns.

This would entail adding a field to `agg_stats_to_calculate`

```python
 Unknown: [ "nunique"],
```

### PR DESCRIPTION
**Pull request by mingdavidqi**
- Calculate nunique for Unknown columns in `_get_describe_dict`
- Closes #1257

### CODE REVIEW

File: woodwork/column_schema.py [hunk 1]
```diff
@@ -152,6 +152,11 @@ def is_boolean(self):
         ltype_class = type(self.logical_type)
         return ltype_class == Boolean or ltype_class == BooleanNullable
 
+    @property
+    def is_unknown(self):
+        """Whether the ColumnSchema is a Datetime column"""
```
Comment (tamargrey) [inline woodwork/column_schema.py:157]: docstring needs updating!

File: woodwork/statistics_utils/_get_describe_dict.py [hunk 2]
```diff
@@ -62,6 +62,7 @@ def _get_describe_dict(
         "category": ["count", "nunique"],
         "numeric": ["count", "max", "min", "nunique", "mean", "std"],
         Datetime: ["count", "max", "min", "nunique", "mean"],
+        Unknown: ["nunique"],
```
Comment (tamargrey) [inline woodwork/statistics_utils/_get_describe_dict.py:65]: Need to add `"count "` or Unknown columns will no longer get count calculated for them (see line 122 where all other columns get `"count"` calculated for them

File: woodwork/column_schema.py (hunk 1 above)
Comment (mingdavidqi) [reply woodwork/column_schema.py]: updated

File: woodwork/statistics_utils/_get_describe_dict.py (hunk 2 above)
Comment (mingdavidqi) [reply woodwork/statistics_utils/_get_describe_dict.py]: added

**Comment by tamargrey** [comment]
I don't think we currently test `describe` on any Unknown columns, which is why these changes haven't broken any tests. Can we add a test that explicitly sets a logical type as Unknown before running `describe` so we can confirm that both `count` and `nunique` are present?

**Review by tamargrey — APPROVED** [review]
lgtm!

**Pull request merged**